projects/usrpe31x/system_bd.tcl

```diff
@@ -0,0 +1,303 @@
+# create board design
+# default ports
+
+create_bd_intf_port -mode Master -vlnv xilinx.com:interface:ddrx_rtl:1.0 ddr
+create_bd_intf_port -mode Master -vlnv xilinx.com:display_processing_system7:fixedio_rtl:1.0 fixed_io
+
+create_bd_port -dir O spi0_csn_0
+create_bd_port -dir O spi0_csn_1
+create_bd_port -dir O spi0_csn_2
+create_bd_port -dir O spi0_clk
+create_bd_port -dir O spi0_mosi
+create_bd_port -dir I spi0_miso
+
+create_bd_port -dir I spi1_csn
+create_bd_port -dir I spi1_clk
+create_bd_port -dir I spi1_mosi
+create_bd_port -dir O spi1_miso
+
+create_bd_port -dir I -from 63 -to 0 ps_gpio_i
+create_bd_port -dir O -from 63 -to 0 ps_gpio_o
+create_bd_port -dir O -from 63 -to 0 ps_gpio_t
+
+# interrupts
+
+create_bd_port -dir I -type intr ps_intr_00
+create_bd_port -dir I -type intr ps_intr_01
+create_bd_port -dir I -type intr ps_intr_02
+create_bd_port -dir I -type intr ps_intr_03
+create_bd_port -dir I -type intr ps_intr_04
+create_bd_port -dir I -type intr ps_intr_05
+create_bd_port -dir I -type intr ps_intr_06
+create_bd_port -dir I -type intr ps_intr_07
+create_bd_port -dir I -type intr ps_intr_08
+create_bd_port -dir I -type intr ps_intr_09
+create_bd_port -dir I -type intr ps_intr_10
+create_bd_port -dir I -type intr ps_intr_11
+create_bd_port -dir I -type intr ps_intr_12
+create_bd_port -dir I -type intr ps_intr_13
+create_bd_port -dir I -type intr ps_intr_14
+create_bd_port -dir I -type intr ps_intr_15
```

Comment:
Considering all the ps_intr ports are connected to 1'b0 in the system_top.v, can we just remove them?